k8/deployment.yaml

```diff
@@ -39,7 +39,7 @@ spec:
     spec:
       containers:
       - name: polaris
-        image: localhost:5001/polaris:latest
+        image: localhost:5001/apache/polaris:latest
```

Comment:
just wondering: why do we need `localhost:5001`? Is it not assumed by default?

Reply:
No, that's the registry part of the image name, and defaults to `docker.io` if unspecified: `apache/polaris` expands to `docker.io/apache/polaris`.

[This post](https://blog.nathanv.me/posts/docker-image-names-for-dummies/) explains it well.